fcrepo-client-impl/src/main/java/org/fcrepo/client/impl/FedoraObjectImpl.java

```diff
@@ -80,4 +89,32 @@ public FedoraObjectImpl(final FedoraRepository repository, final HttpHelper http
         }
         return set;
     }
+
+    @Override
+    public FedoraObject createObject() throws FedoraException {
+        final HttpPost post = httpHelper.createPostMethod(getPath(), null);
+        try {
+            final HttpResponse response = httpHelper.execute(post);
+            final String uri = post.getURI().toString();
+            final StatusLine status = response.getStatusLine();
+            final int statusCode = status.getStatusCode();
+
+            if (statusCode == SC_CREATED) {
+                return repository.getObject(response.getFirstHeader("Location").getValue().substring(
+                        repository.getRepositoryUrl().length()));
+            } else if (statusCode == SC_FORBIDDEN) {
+                LOGGER.error("request to create resource {} is not authorized.", uri);
+                throw new ForbiddenException("request to create resource " + uri + " is not authorized.");
+            } else {
+                LOGGER.error("error creating resource {}: {} {}", uri, statusCode, status.getReasonPhrase());
```

Comment:
Log message is different than the exception message: "creating" vs. "retrieving".